src/sentry/conf/server.py

```diff
@@ -502,6 +502,9 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
 CELERY_CREATE_MISSING_QUEUES = True
 CELERY_REDIRECT_STDOUTS = False
 CELERYD_HIJACK_ROOT_LOGGER = False
+CELERY_TASK_SERIALIZER = "pickle"
+CELERY_RESULT_SERIALIZER = "pickle"
+CELERY_ACCEPT_CONTENT = {"pickle", "json"}
```

Comment:
Shouldn't this just be `{"pickle"}`, or am I missing something?

Reply:
Agree with @joshuarli here, I think it should just be `{"pickle"}`

Reply:
I think it's fine to set this to both. We could have new tasks that specify to use the json format, and this would allow celery to accept and process that task. It shouldn't cause any harm.

https://docs.celeryproject.org/en/stable/userguide/configuration.html#std:setting-accept_content

Reply:
Whenever a new task is added to use the json format, we could add that on. I'd usually err on the side of caution when the change is very easy to make - it's also of course being lazy.